app/scripts/controllers/pod.js

```diff
@@ -204,54 +205,47 @@ angular.module('openshiftConsole')
       });
     };
 
+    var podResolved = function(pod, action) {
+      $scope.loaded = true;
+      $scope.pod = pod;
+      var pods = {};
+      pods[pod.metadata.name] = pod;
+      setLogVars(pod);
+      setContainerVars();
+      ImageStreamResolver.fetchReferencedImageStreamImages(pods, $scope.imagesByDockerReference, $scope.imageStreamImageRefByDockerReference, requestContext);
+      $scope.containerTerminals = makeTerminals();
```

Comment:
not sure this is a safe update, you've changed it so its now always calling makeTerminals on every update of the pod